src/dec_png.c

```diff
@@ -86,112 +86,112 @@ int SUB_NAME(unsigned char *pngbuf,g2int *width,g2int *height,char *cout)
+** create and initialize png_structs
 */
 
-    png_ptr = png_create_read_struct(PNG_LIBPNG_VER_STRING, (png_voidp)NULL, 
-                                      NULL, NULL);
+    png_ptr = png_create_read_struct(PNG_LIBPNG_VER_STRING, (png_voidp)NULL,
+                                     NULL, NULL);
     if (!png_ptr)
-       return (-1);
+        return (-1);
 
     info_ptr = png_create_info_struct(png_ptr);
     if (!info_ptr)
     {
-       png_destroy_read_struct(&png_ptr,(png_infopp)NULL,(png_infopp)NULL);
-       return (-2);
+        png_destroy_read_struct(&png_ptr,(png_infopp)NULL,(png_infopp)NULL);
+        return (-2);
     }
 
     end_info = png_create_info_struct(png_ptr);
     if (!end_info)
     {
-       png_destroy_read_struct(&png_ptr,(png_infopp)info_ptr,(png_infopp)NULL);
-       return (-2);
+        png_destroy_read_struct(&png_ptr,(png_infopp)info_ptr,(png_infopp)NULL);
+        return (-2);
     }
 
-/* 
-**     Set Error callback  
+/*
+**     Set Error callback
 */
 
     if (setjmp(png_jmpbuf(png_ptr)))
     {
-       png_destroy_read_struct(&png_ptr, &info_ptr,&end_info);
-       return (-3);
+        png_destroy_read_struct(&png_ptr, &info_ptr,&end_info);
+        return (-3);
     }
 
 /*
-**    Initialize info for reading PNG stream from memory 
+**    Initialize info for reading PNG stream from memory
 */
 
     read_io_ptr.stream_ptr=(png_voidp)pngbuf;
     read_io_ptr.stream_len=0;
 
 /*
-**    Set new custom read function 
+**    Set new custom read function
 */
 
     png_set_read_fn(png_ptr,(png_voidp)&read_io_ptr,(png_rw_ptr)user_read_data);
 
 /*     png_init_io(png_ptr, fptr); */
 
 /*
-**     Read and decode PNG stream 
+**     Read and decode PNG stream
 */
 
     png_read_png(png_ptr, info_ptr, PNG_TRANSFORM_IDENTITY, NULL);
 
 /*
-**     Get pointer to each row of image data 
+**     Get pointer to each row of image data
 */
 
     row_pointers = png_get_rows(png_ptr, info_ptr);
 
 /*
-**      Get image info, such as size, depth, colortype, etc...  
+**      Get image info, such as size, depth, colortype, etc...
 */
 
 /*    printf("SAGT:png %d %d %d\n",info_ptr->width,info_ptr->height,info_ptr->bit_depth);*/
 /*    (void)png_get_IHDR(png_ptr, info_ptr, (png_uint_32 *)width, (png_uint_32 *)height,
-               &bit_depth, &color, &interlace, &compres, &filter);*/
+      &bit_depth, &color, &interlace, &compres, &filter);*/
     (void)png_get_IHDR(png_ptr, info_ptr, &w32, &h32,
-               &bit_depth, &color, &interlace, &compres, &filter);
+                       &bit_depth, &color, &interlace, &compres, &filter);
 
     *height = h32;
     *width = w32;
 
 /*
-**     Check if image was grayscale  
+**     Check if image was grayscale
 */
 
 /*
-    if (color != PNG_COLOR_TYPE_GRAY ) {
-       fprintf(stderr,"dec_png: Grayscale image was expected. \n");
-    }
+  if (color != PNG_COLOR_TYPE_GRAY ) {
+  fprintf(stderr,"dec_png: Grayscale image was expected. \n");
```

Comment:
This should be indented.

Reply:
Who are we to question emacs auto-indent function?

In this case it is not indented because it is commented out code. Them's the breaks...